indent/tsx.vim

```diff
@@ -1,4 +1,3 @@
-exe 'runtime! indent/typescript.vim'
 " Save the current JavaScript indentexpr.
```

Comment:
Is deleting typescript indentation file safe?

Reply:
Yes, I've been using this and it works fine. That's the magic of `typescript.tsx`, it includes both.
I'll add some tests for tsx + non tsx combined.

Reply:
Cool